cgi-bin/ljtestlib.pl

```diff
@@ -0,0 +1,83 @@
+package LJ;
```

Comment:
This is a neat idea, but what do you think about putting this file in the `t/` directory? I think my concern is having "test libraries" in the same directory as production libraries, and wondering if that can cause problems down the road.

Reply:
agreed; I'll do that.